app/src/main/java/me/saket/dank/widgets/IndentedLayout.java

```diff
@@ -10,41 +10,82 @@
 import androidx.annotation.IntRange;
 import androidx.annotation.Nullable;
 import androidx.annotation.Px;
+import com.f2prateek.rx.preferences2.Preference;
+import com.jakewharton.rxbinding2.view.RxView;
+
 import android.util.AttributeSet;
 import android.util.TypedValue;
 import android.widget.LinearLayout;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
 import me.saket.dank.R;
+import me.saket.dank.di.Dank;
 
 public class IndentedLayout extends LinearLayout {
-
-  private static final int DEFAULT_SPACING_PER_DEPTH_DP = 8;
-  private static final int DEFAULT_LINE_WIDTH = 4;
+  private static final int DEFAULT_SPACING_PER_DEPTH_DP = 10;
+  private static final int DEFAULT_LINE_WIDTH = 6;
 
   private int indentationDepth;
   private final int spacePerDepthPx;
   private Paint indentationLinePaint;
-  private Path indentationLinePath;
+  private int indentationLineWidth;
+  private int defaultIndentationLineColor;
 
   private int originalPaddingStart;
 
+  private static final String[] INDENTATION_COLORS = {
```

Comment:
I've just got an idea that we can move this to resources as array which would contain `<item>@color/name_of_color</item>` in it. That way we could reuse existing colors or take these from `colors_material.xml`.

It can be read via `getResources().getIntArray(R.array.some_name)` then.